pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -153,6 +153,31 @@ func Test_EventListenerValidate(t *testing.T) {
 				),
 				bldr.EventListenerTrigger("tb", "tt", "v1alpha1"))),
 		raiseErr: false,
+	}, {
+		name: "Valid EventListener Multiple Interceptor",
+		el: bldr.EventListener("name", "namespace",
+			bldr.EventListenerSpec(
+				bldr.EventListenerTrigger("tb", "tt", "v1alpha1",
+					bldr.EventListenerTriggerInterceptor("svc", "v1", "Service", "namespace"),
+					bldr.EventListenerTriggerInterceptor("svc", "v1", "Service", "namespace")))),
+		raiseErr: false,
+	}, {
+		name: "Both Interceptor and Interceptors Present",
+		el: &v1alpha1.EventListener{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "name",
+				Namespace: "namespace",
+			},
+			Spec: v1alpha1.EventListenerSpec{
+				Triggers: []v1alpha1.EventListenerTrigger{{
+					Bindings:              []*v1alpha1.EventListenerBinding{{Name: "tb"}},
+					Template:              v1alpha1.EventListenerTemplate{Name: "tt"},
+					DeprecatedInterceptor: &v1alpha1.EventInterceptor{},
+					Interceptors:          []*v1alpha1.EventInterceptor{{}, {}},
+				}},
+			},
+		},
+		raiseErr: true,
```

Comment:
if you feel motivated, i find it much easier to maintain when there is one test for success cases and a separate test for error cases - much easier to see what is expected to be valid and what isnt